go.mod

```
@@ -2,7 +2,7 @@ module github.com/kopia/kopia
 
 go 1.23.0
 
-toolchain go1.23.7
+toolchain go1.24.1
```

@dependabot WTF? why did it change this line???

```suggestion
toolchain go1.23.7
```